frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx

```diff
@@ -66,6 +66,9 @@ export const IconsComponent = ({
     state => state.isUserAuthorized.data
   );
   const { role } = useTypedSelector(state => state.isUserAuthorized.data);
+  const { role: OtherUserId } = useTypedSelector(
+    state => state.otherUserData.data
+  );
```

Comment:
why is the value of property 'role' assigned in the 'id' variable?

Reply:
should be renamed to 'otherUserRole' or smthg

Reply:
i agree with you - done